src/ml/mlApi.ts

```diff
@@ -14,3 +15,17 @@ export async function deleteModel(projectId: string, modelId: string): Promise<v
     origin: api.mlOrigin,
   });
 }
+
+/**
+ * Get a Firebase model.
+ * @param projectId The project where the model exists
+ * @param modelId The id of the model to get
+ * @return { Promise<FirebaseModel> } The model
```

Comment:
Types don't need to be specified (since it's typescript, the signature of the function is sufficient - which generally includes return types too)
```suggestion
 * @return The model
```

Reply:
done